fastai/vision/learner.py

```diff
@@ -19,13 +19,15 @@ def _squeezenet_split(m:nn.Module): return (m[0][0][5], m[0][0][8], m[1])
 def _densenet_split(m:nn.Module): return (m[0][0][7],m[1])
 def _vgg_split(m:nn.Module): return (m[0][0][22],m[1])
 def _alexnet_split(m:nn.Module): return (m[0][0][6],m[1])
+def _mobilenetv2_split(m:nn.Module): return (m[0][0][10],m[1])
```

Comment:
This was a somewhat arbitrary choice, but it splits the network into roughly equal parts

Reply:
Sounds fine to me!